addons/mass_mailing_sms/models/mailing_trace.py

```diff
@@ -30,6 +30,7 @@ class MailingTrace(models.Model):
         ('sms_number_format', 'Wrong Number Format'),
         ('sms_credit', 'Insufficient Credit'),
         ('sms_server', 'Server Error'),
+        ('sms_unregistered', 'Unregistered Account'),
```

Comment:
We plan to clean and reduce failure_type size ... notably try to shorten key length to save db space. For example our production has several M traces, a few characters help saving storage :) . So maybe sms_acc ? We know it is linked to the account.